File: packages/urdu-js/tests/analysis/rtl.test.ts
import { describe, it, expect } from 'vitest'
import { isRTL } from '../../src/analysis/rtl.js'

describe('isRTL', () => {
  it('empty → false', () => expect(isRTL('')).toBe(false))
  it('Urdu is RTL', () => expect(isRTL('پاکستان')).toBe(true))
  it('Arabic is RTL', () => expect(isRTL('مرحبا')).toBe(true))
  it('Latin is LTR', () => expect(isRTL('Hello')).toBe(false))
  it('digits alone → false', () => expect(isRTL('123')).toBe(false))
})

File: packages/urdu-dart/test/analysis/rtl_test.dart
import 'package:test/test.dart';
import 'package:urdu_tools/urdu_tools.dart';

void main() {
  group('isRTL', () {
    test('empty -> false', () => expect(isRTL(''), isFalse));
    test('Urdu is RTL', () => expect(isRTL('پاکستان'), isTrue));
    test('Arabic is RTL', () => expect(isRTL('مرحبا'), isTrue));
    test('Latin is LTR', () => expect(isRTL('Hello'), isFalse));
    test('digits alone -> false', () => expect(isRTL('123'), isFalse));
  });
}
